lightning/src/routing/router.rs

```diff
@@ -173,6 +173,9 @@ impl_writeable_tlv_based!(RouteParameters, {
 	(4, final_cltv_expiry_delta, required),
 });
 
+/// Maximum total CTLV difference we allow for a full payment path
```

Comment:
nit: end doc with period.

Reply:
Done in https://github.com/lightningdevkit/rust-lightning/pull/1234/commits/ebecbddf9c7375ddea65b0263e959b60eb308b6d.